app/url_mappings.go

```diff
@@ -11,6 +11,7 @@ func mapUrls() {
 	router.GET("/deck/:deckID", decks.GetDeck)
 	router.POST("/deck", decks.CreateDeck)
 	router.DELETE("/deck/:deckID", decks.DeleteDeck)
+	router.DELETE("/deck/:deckID/card/:cardID", decks.DeleteCardFromDeck)
```

Comment:
Maybe we need to discuss whether we should using `/:cardID` to delete (or add) one card ID at a time.